src/index.ts

```diff
@@ -53,12 +53,7 @@ class MongooseTsgen extends Command {
   };
 
   // path of mongoose models folder
-  static args = [
-    {
-      name: "root-path",
-      default: "."
-    }
-  ];
+  static args = [{ name: "root-path" }];
```

Comment:
What happened to the default value?

Reply:
Since the new models path resolution logic needs to differiate betwen user input (search user input path directly) and no input (search recursively for `models` folder), adding a default value removes the ability to determine this.